src/components/ConfirmButton/ConfirmButton.tsx

```diff
@@ -0,0 +1,148 @@
+import { buttonMessages, commonMessages } from "@dashboard/intl";
+import CircularProgress from "@material-ui/core/CircularProgress";
```

Comment:
**suggestion**: maybe we should create a story in MacawUI for adding `CircularProgress` and `CheckIcon`icons?

Reply:
IMO we should if we want to completely eliminate mui